packages/api/src/controllers/asset.ts

```diff
@@ -110,6 +111,24 @@ function withPlaybackUrls(ingest: string, asset: WithID<Asset>): WithID<Asset> {
   };
 }
 
+export async function createAsset(asset: WithID<Asset>, queue: Queue) {
+  asset = await db.asset.create(asset);
+  await queue.publishWebhook("events.asset.created", {
+    type: "webhook_event",
+    id: uuid(),
+    timestamp: asset.createdAt,
+    event: "asset.created",
```

Comment:
They already are! The allowed values are enumerated [here](https://github.com/livepeer/livepeer-com/blob/1c212c6a02e707e66da138e373672ff6c7e77bf8/packages/api/src/schema/schema.yaml#L202).

Also everywhere we create any field that should represent these values we make a reference to this field, which is aliased [here](https://github.com/livepeer/livepeer-com/blob/ee0c2199d465c0f4d31554fe12a6096deffa9331/packages/api/src/store/webhook-table.ts#L7) as `EventKey`.

Finally, if we try to set anything here that is not one of the enum values, it results in a Typescript error 😄 
![Screen Shot 2022-06-07 at 15 20 12](https://user-images.githubusercontent.com/1613383/172454643-27d81049-7a3c-42ef-8923-be0fa4f63f99.png)

So I don't think it's worth repeating those enum values as "typescript enums" instead of a string literal union, as that would just obly us to keep those 2 separate declarations in sync. The string literal union also allows for cool things like [this](https://github.com/livepeer/livepeer-com/blob/5b4b5440c9d1a6bf8dc8542b5f2c45639d55771f/packages/api/src/store/queue.ts#L23), so I think they are actually better than the enums in some sense!